www/src/pages/en/introduction.md

```diff
@@ -8,9 +8,9 @@ layout: ../../layouts/docs.astro
 <iframe width="560" height="315" src="https://www.youtube.com/embed/PbjHxIuHduU" title="The best stack for your next project" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </div>
 
-## What is the T3 Stack?
+## The T3 Stack
 
-The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t3dotgg) focused on simplicity, modularity, and full-stack typesafety. It consists of:
+The _"T3 Stack"_ is a web development stack made by [Theo](https://twitter.com/t3dotgg) focused on simplicity, modularity, and full-stack typesafety. The core pieces are...
```

Comment:
A colon instead of ... maybe? Doesn't matter too much though

Reply:
I lightly prefer the `...` as it reads smoother. Tbh not sure how I feel about a list here, considering making it an inline comma-separated sentence